src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { lazy, useCallback, useEffect } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { Checkbox, FormControlLabel, Grid } from '@mui/material';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useNavigate } from 'react-router';
+import { useDispatch } from 'react-redux';
+import Button from '@mui/material/Button';
+import { useFetchSigInMutation } from '../../services/AuthServices';
+import { IErrorResponse } from '../../models/IErrorResponse';
+import { ISignInParams } from '../../models/ISignInParams';
+import { schemaLogin } from './schema';
+
+import '../../styles/auth.scss';
+
+import {
+  InputLabel,
+  InputName, InputType,
+  MESSAGES_TEXT,
+  RouterLinks,
+  RouterLinksName,
+  TYPES_ALERT
+} from '../../constants/constants';
+
+import { IAlertTypeProps, showAlert } from '../../store/reducers/AlertSlice';
+import useToggleVisibility from '../../hooks/useToggleVisibility';
+
+const TextField = lazy(() => import(/* webpackChunkName: "TextField" */ '../../components/TextField/TextField'))
+const Loader = lazy(() => import(/* webpackChunkName: "Loader" */ '../../components/Loader/Loader'))
+
+const Login = () => {
+  const methods = useForm<ISignInParams>({
+    defaultValues: {
+      [InputName.login]: '',
+      [InputName.password]: ''
+    },
+    mode: 'onBlur',
+    resolver: yupResolver(schemaLogin)
+  });
+
+  const [fetchLogin, {
+    isLoading,
+    data,
+    error,
+    isSuccess,
+    isError
+  }] = useFetchSigInMutation();
```

Comment:
❗️
Проверь весь пр на предмет таких опечаток
```suggestion
  }] = useFetchSignInMutation();
```

Reply:
готово